_it/tour/tuples.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: tour
+title: Tuples
+partof: scala-tour
+
+num: 8
+next-page: mixin-class-composition
+previous-page: traits
+topics: tuples
+
+redirect_from: "/tutorials/tour/tuples.html"
+language: IT
+---
+Traduzione a cura di: Andrea Mucciarelli (https://github.com/IowU)
+
+
+In Scala, una tupla è un valore che contiene un numero fisso di elementi, ognuno
+con il proprio tipo. Le tuple sono immutabili.
+
+Le tuple sono particolarmente utili per restituire valori multipli da un methodo.
+
+Una tupla con due elementi può essere creata come segue:
+
+```tut
+val ingredient = ("Sugar" , 25)
+```
+
+Questo creerà una tupla contenente un elemento di tipo `String` e uno di tipo `Int`.
+
+Il tipo inferito di `ingredient` è `(String, Int)`, che è un'abbreviazione per `Tuple2[String, Int]`.
+
+Per rappresentare le tuple, scala una serie di classi: `Tuple2`, `Tuple3`, ecc., fino a `Tuple22` (compreso).
```

Comment:
"ha"?  
```suggestion
Per rappresentare le tuple, scala ha una serie di classi: `Tuple2`, `Tuple3`, ecc., fino a `Tuple22` (compreso).
```

Reply:
I was thinking "usa una serie ...", and I forgot a word. Your suggestion makes more sense though.